pic-sure-hpds-ui/src/main/webapp/picsureui/common/styles.js

```diff
@@ -3,6 +3,6 @@ define(["jquery", "text!styles.css", "text!bootstrapStyles", "text!treeviewStyle
 	$('head').append("<style></style>");
 	$('head style').html(
 			bootstrapStyles.replace(new RegExp('\.\./fonts/', 'g'),
-					'webjars/bootstrap/3.4.1/fonts/')
+					'/picsureui/webjars/bootstrap/3.4.1/fonts/')
```

Comment:
This was not working on /psamaui paths, which was not a problem until we started using pushstate instead of redirecting from login to other pages